bin/extract-ORFs.py

```diff
@@ -39,20 +39,35 @@
 
     parser.add_argument(
         '--minORFLength', metavar='LEN', type=int, default=None,
-        help='Only ORFs of at least this length will be written to stdout.')
+        help=('Only ORFs of at least this length will be written to stdout.'))
 
     parser.add_argument(
         '--maxORFLength', metavar='LEN', type=int, default=None,
-        help='Only ORFs of a maximum of this length will be written to'
-        'stdout.')
+        help=('Only ORFs of a maximum of this length will be written to '
+              'stdout.'))
+
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument(
+        '--kozakOnly', default=False, action='store_true',
+        help=('Only ORFs that also have a Kozak consensus will be written to '
+              'stdout. A file with Kozak consensus information will be '
+              'written containing only the Kozak consensus sequences that '
+              'correspond to an ORF. Only applicable if DNA reads given.'))
+
+    group.add_argument(
+        '--kozakInfoFile', type=str, default=None,
```

Comment:
You don't need `default=None`. That is already the default default (unless you tell the option parser to use something else when you create it).

Reply:
And you don't need `type=str` as it's also the default.

Reply:
ah right. done